src/Transformer/YamlTransformer.php

```diff
@@ -135,6 +136,9 @@ private function checkForDeprecatedConfig(array $document, MutableConfigCollecti
         if (!($collection instanceof MemoryConfigCollection)) {
             return;
         }
+        if ($document['inSupportedModule'] && !Deprecation::getShowNoticesCalledFromSupportedCode()) {
+            return;
+        }
```

Comment:
Need class_exists check

Reply:
Also either need method_exists check like below, or else a new `conflicts` key in composer.json for older versions of framework.

Reply:
Updated

Reply:
Not updated?

Reply:
Forgot to push :/